src/background.js

```diff
@@ -1,7 +1,7 @@
 // Param values from https://developer.mozilla.org/Add-ons/WebExtensions/API/contextualIdentities/create
 const FACEBOOK_CONTAINER_NAME = "Facebook";
-const FACEBOOK_CONTAINER_COLOR = "blue";
-const FACEBOOK_CONTAINER_ICON = "briefcase";
+const FACEBOOK_CONTAINER_COLOR = "toolbar";
+const FACEBOOK_CONTAINER_ICON = "fence";
```

Comment:
Since the fence icon and toolbar color are only available with Firefox 67 ([~~Bug 1532746~~](https://bugzilla.mozilla.org/show_bug.cgi?id=1532746)), we'd need to either set the `strict_min_version` in the manifest.json to `67.0` - or, preferably, add something along the lines of

```suggestion
const is67OrLater = parseInt(browser.runtime.getBrowserInfo().version) >= 67;
const FACEBOOK_CONTAINER_COLOR = is67OrLater ? "toolbar" : "blue";
const FACEBOOK_CONTAINER_ICON = is67OrLater ? "fence" : "briefcase";
```

Otherwise the Add-on will fail to initialize correctly on Firefox <67.

Reply:
Great catch!

Reply:
Dang, that's an async API so I'll have to do this in the `setupContainer` function. Not as clean as the const definitions up here. 🤷‍♂